content/en/user-guide/chaos-engineering/chaos-plugin/index.md

```diff
@@ -0,0 +1,181 @@
+This strategy helps gauge the effectiveness of the system's deployment procedures and its resilience against infrastructure disruptions, which is a key element of chaos engineering.
+
+You can use LocalStack Chaos Plugin to cause API failures for following or any combination thereof:
+- Service
+- Region
+- Operation
+
+You can customise the HTTP error code and message that LocalStack responds with.
+If required, you can make the failures occur probabilistically.
+
+Furthermore, the Chaos Plugin can also be configured to add a network latency for all calls.
+
+{{< alert title="Note">}}
+Chaos Plugin is available as part of the LocalStack Enterprise plan.
+If you'd like to try it out, please [contact us](https://www.localstack.cloud/demo) to request access.
+{{< /alert >}}
+
+## Prerequisites
+
+The prerequisites for this guide are:
+
+- LocalStack Pro with [LocalStack CLI](https://docs.localstack.cloud/getting-started/installation/#localstack-cli) & [LocalStack Auth Token](https://docs.localstack.cloud/getting-started/auth-token/)
+- [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/)
+- [Python](https://www.python.org/downloads/)
+
+
+## Configuration
+
+The chaos effects supported by Chaos Plugin are broadly categorised into two groups.
+**Faults** lead to an application-level HTTP error, and **Network Effects** introduce network-level effects to the connections.
+
+### Faults
+
+Faults can be configured using the endpoint at `/_localstack/chaos/faults`.
+The configuration schema consists of an array of one or more rules, where each rule specifies the conditions for the fault to occur.
+When active, rules are evaluated sequentially on every request to LocalStack until the first match.
+
+The schema for the configuration is as follows.
+
+```json
+[
+    {
+        "region": "(str) Region name, e.g. 'ap-south-1'. If omitted, all regions are affected.",
+        "service": "(str) Name of the service, e.g. 'kinesis'. If omitted, all services are affected.",
+        "operation": "(str) Name of the operation, e.g. 'PutRecord'. If omitted, all operations are affected.",
+        "probability": "(num) Probability of invoking this rule, e.g. 0.5. If omitted, 1 is used.",
+        "error": {
+            "statusCode": "(int) HTTP status code to use in response, e.g. 503. If omitted, 503 is used.",
+            "code": "(str) Descriptive error code used in response. If omitted, 'ServiceUnavailable' is used."
+        }
+    },
+    ...
+]
+```
+
+The endpoint allows the following operations:
+- `GET`: Get current configuration
+- `POST`: Add new configuration
+- `PATCH`: Add a rule
+- `DELETE`: Delete a rule
+
+An empty array `[]` disables the faults entirely, while an empty rule in the array `[{}]` causes all AWS operations to lead to faults.
+
+### Network Effects
+
+Network effects are configured using the endpoint `/_localstack/chaos/effects`.
```

Comment:
to me the name `/_localstack/chaos/effects` on its own is not specific enough. its makes sense in the context of the docs here, but I would not know what that is if I just see a call to `chaos/effects` somewhere in my test suite. 

maybe `chaos/network` is a better name for it?

Reply:
'effects' sounded like the right word to me. For the other endpoint, the end result is a fault, similarly the end result here is a network 'effect'.

I'm happy to change it, it might be worth considering a wider opinion from GTM, Devrel, etc.

Reply:
'effect' on its own is a bit too general for me, because everytying is an effect. 'networking-effect' would be better, but still could we be a bit more precise?